README.md

```diff
@@ -16,11 +16,20 @@ python -m src data --download aclImdb
 ```
 
 
-### Macron
+### Twitter
+One can download tweets from twitter, a candidat must be mention:
 
 ```python
-python -m src data --download macron
+python -m src data --download twitter --mention [candidat]
```

Comment:
Add an error -> mention wasn took in considaeration when making twitter API request